bundles/multitenant-managed-api-service/multitenant-managed-api-service.package.yaml

```diff
@@ -0,0 +1,5 @@
+channels:
+  - currentCSV: multitenant-managed-api-service.v1.21.0
+    name: rhmi
+defaultChannel: rhmi
```

Comment:
the channels seems to be different in metadata/annotations.yaml

Reply:
I was basing this off of single tenant RHOAM which has `stable` in [bundles/managed-api-service/1.22.0/metadata/annotations.yaml](https://github.com/integr8ly/integreatly-operator/blob/master/bundles/managed-api-service/1.22.0/metadata/annotations.yaml#L7-L8) but `rhmi` in [bundles/managed-api-service/managed-api-service.package.yaml](https://github.com/integr8ly/integreatly-operator/blob/master/bundles/managed-api-service/managed-api-service.package.yaml#L3-L4)

Should I change the channel to be `stable` in both or `rhmi` in both or keep it as is?